content/movie-review-pt2.md

```diff
@@ -10,8 +10,9 @@
 
 # TL;DR
 
-- Creating an account requires that we calculate the space and rent to allocate for a new account
-- A Program Derived Address (PDA) is derived from a program Id and an optional list of seeds
+- Program state is stored in external accounts rather than in the program itself
```

Comment:
sugg: "another" account, rather than "external" account (which may be interpreted as "off-chain"?)  I am judging myself negatively for even bringing this one up...